ajax/libs/tern/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "tern",
+  "license": "MIT",
+  "version": "0.18.0",
+  "author": "Marijn Haverbeke <marijnh@gmail.com>",
+  "description": "A JavaScript code analyzer for deep, cross-editor language support",
+  "filename": "lib/tern.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/ternjs/tern.git"
+  },
+  "dependencies": {
+    "acorn": "^2.7.0",
+    "enhanced-resolve": "^0.9.1",
+    "glob": "3",
+    "minimatch": "0.2",
+    "typescript": "=1.0.1"
+  },
+  "homepage": "http://ternjs.net/",
+  "keywords": [
+    "stand-alone",
+    "editor-independent",
+    "JavaScript",
+    "analyzer",
+    "integration"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/ternjs/tern.git",
+    "basePath": "",
+    "files": [
+      "lib/(comment*|condense*|def*|infer*|signal*|tern*)+.(js|map)",
+      "plugin/(angular*|commonjs*|complete_strings*|doc_comment*|es_modules*|modules*|node*|node_resolve*|requirejs*|webpack*)+.(js|map)"
```

Comment:
by Tern's [Manual](http://ternjs.net/doc/manual.html) 
>Node.js plugin
The node.js plugin, called "node", provides variables that are part of the node environment, such as process and __dirname, and loads the commonjs and node_resolve plugins to allow node-style module loading. It defines types for the built-in modules that node.js provides ("fs", "http", etc).

>Node.js resolve plugin
This plugin defines the node.js module resolution strategy—things like defaulting to index.js when requiring a directory and searching node_modules directories. It depends on the modules plugin. Note that this plugin only does something meaningful when the Tern server is running on node.js itself.


yes, i think they are for node, since we don't host the type of files, do we just close this PR and issue?